src/adapter/sources.ts

```diff
@@ -68,6 +69,35 @@ const defaultTimeouts: SourceMapTimeouts = {
   output: 1000,
 };
 
+export class UrlToSourceMap {
```

Comment:
I'd recommend using our `MapUsingProjection` here, I think that does what you're after